src/components/swap/SwapBuyFiatButton.tsx

```diff
@@ -135,7 +128,6 @@ export default function SwapBuyFiatButton() {
       >
         <StyledTextButton onClick={handleBuyCrypto} disabled={buyCryptoButtonDisabled} data-testid="buy-fiat-button">
           <Trans>Buy</Trans>
-          {!buyFiatFlowCompleted && <Dot data-testid="buy-fiat-flow-incomplete-indicator" />}
```

Comment:
are you able to remove the hook now? useBuyDiatFlowCompleted?

unclear if it's used elsewhere

Reply:
Believe so, just checked and the hook isn't used anywhere else!